GitFyle.Core.Api.Tests.Unit/Controllers/Contributions/ContributionsControllerTests.Exceptions.Delete.cs

```diff
@@ -0,0 +1,168 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using GitFyle.Core.Api.Models.Foundations.Contributions;
+using GitFyle.Core.Api.Models.Foundations.Contributions.Exceptions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using RESTFulSense.Clients.Extensions;
+using RESTFulSense.Models;
+using Xeptions;
+
+namespace GitFyle.Core.Api.Tests.Unit.Controllers.Contributions
+{
+    public partial class ContributionsControllerTests
+    {
+        [Theory]
+        [MemberData(nameof(ValidationExceptions))]
+        public async Task ShouldReturnBadRequestOnDeleteIfValidationErrorOccursAsync(
+                Xeption validationException)
+        {
+            // given
+            Guid someId = Guid.NewGuid();
```

Comment:
```suggestion
            Guid someContributionId = Guid.NewGuid();
```
fix name here as well